### PR TITLE
Added new module

### PR DESCRIPTION
**Pull request by aleksvagachev**
##### SUMMARY
Added a new module cockroachdb_db.py

##### ISSUE TYPE
- New Module Pull Request

##### COMPONENT NAME
Create, modify or delete a CockroachDB database

### CODE REVIEW

File: plugins/modules/cockroachdb_db.py [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: cockroachdb_db
+short_description: Create, modify or delete a CockroachDB database
+description:
+  - Creates, modifies or deletes a CockroachDB database.
+version_added: '0.3.0'
+author:
```
Comment (Andersson007) [inline plugins/modules/cockroachdb_db.py:17]:
```suggestion
author:
  - Aleks Vagachev (@aleksvagachev)
```

**Comment by aleksvagachev** [comment]
@Andersson007 Hi. Please take a look. Maybe there are some shortcomings.

File: plugins/modules/cockroachdb_db.py [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+        executed_statements.append((query, ()))
+
+    def rename(self, new_name):
+        if self.module.check_mode:
+            return True
+
+        query = 'ALTER DATABASE "%s" RENAME TO "%s"' % (self.name, new_name)
+        self.cursor.execute(query)
+        executed_statements.append((query, ()))
+
```
Comment (Andersson007) [inline plugins/modules/cockroachdb_db.py:149]:
```suggestion
```

It imo doesn't help idempotancy:) I suggest getting rid of this

File: plugins/modules/cockroachdb_db.py [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+    def __change_owner(self, new_owner):
+        query = 'ALTER DATABASE "%s" OWNER TO %s' % (self.name, new_owner)
+        self.cursor.execute(query)
+        executed_statements.append((query, ()))
+
+    def __change_name(self, new_name):
+        query = 'ALTER DATABASE "%s" RENAME TO "%s"' % (self.name, new_name)
+        self.cursor.execute(query)
+        executed_statements.append((query, ()))
```
Comment (Andersson007) [inline plugins/modules/cockroachdb_db.py:175]:
```suggestion
```

File: plugins/modules/cockroachdb_db.py [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+    login_host: 192.168.0.10
+    login_db: acme
+    owner: test
+    name: test_db
+    primary_region: us-east1
+    survive_failure: region
+    regions:
+        - us-east1
+        - us-west1
+        - us-central1
```
Comment (Andersson007) [inline plugins/modules/cockroachdb_db.py:67]:
```suggestion
    primary_region: us-east1
    survive_failure: region
    regions:
        - us-east1
        - us-west1
        - us-central1
```
is it supported?

**Comment by aleksvagachev** [comment]
@Andersson007 Corrected all the comments.

**Review by Andersson007 — APPROVED** [review]

**Comment by Andersson007** [comment]
@aleksvagachev
- LGTM, thanks
- I've just invited you with `maintainer` to this repo, please check your email
- After you accept the invite, please click the `Squash and merge` green button (if later when you have permissions in other repos, if there are more options than Squash and merge, always choose Squash and merge, fyi).

**Pull request merged**